src/Lesson1/Lesson1.java

```diff
@@ -0,0 +1,48 @@
+package Lesson1;
+
+public class Lesson1 {
+    public static void main(String[] args) {
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+    public static void printThreeWords() {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+    public static void checkSumSign(){
+        int a = -5;
+        int b = 14;
+        int c = a + b;
+        if (c >= 0) {
+            System.out.println("Сумма положительная");
+        }
+        if (c < 0) {
+            System.out.println("Сумма отрицательная");
+        }
```

Comment:
```suggestion
        if (c >= 0) {
            System.out.println("Сумма положительная");
        } else {
            System.out.println("Сумма отрицательная");
        }
```
так как условия взаимоисключающие